.github/ISSUE_TEMPLATE/avm_module_issue.yml

```diff
@@ -86,6 +86,7 @@ body:
         - "avm/res/app/job"
         - "avm/res/app/managed-environment"
         - "avm/res/app/session-pool"
+        - "avm/res/authorization/role-assignment"
```

Comment:
Should we remove - "avm/ptn/authorization/role-assignment"?

Reply:
I originally suggested the same, but @jtracey93 wants to keep that module and assumed ownership of the same

Reply:
Isn't the old module intended to be deprecated? I'll add a point to address offline to not block the pr

Reply:
In general yes, it's really just in this case that it so happens that @jtracey93 wants to keep the `ptn` in addition to the `res` module(s)